ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json

```diff
@@ -17,6 +17,8 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "deceasedForenames", "Label": "First name(s)", "HintText": "Include all middle names", "FieldType": "Text", "SecurityClassification": "Public", "Min": 1},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "deceasedSurname", "Label": "Last name(s)", "FieldType": "Text", "SecurityClassification": "Public", "Min": 1},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "deceasedAnyOtherNames", "Label": "Did the deceased have assets in any other names?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "deceasedForeignDeathCertInEnglish", "Label": "Is the original foreign death certificate in english?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "deceasedForeignDeathCertTranslation", "Label": "Is the original foreign death certificate in english?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "deceasedAliasNameList", "Label": "Deceased Alias Names", "FieldType": "Collection", "FieldTypeParameter": "ProbateAliasName", "SecurityClassification": "Public"},
```

Comment:
Is the LiveFrom date ok here? I assume Live means went to Production? Shouldn't this be a more recent date? Or is the feature not used?